src/hist/basehist.py

```diff
@@ -375,7 +374,9 @@ def density(self):
         """
         # ToDo: maybe should not be filled
         # ToDo: flow should be removed
-        return self.view() / self.sum() / np.prod(self.axes.widths, axis=0)
```

Comment:
`reduce` is a better way to write this - and the `prod` is also broken in 0.10.0. I have a PR that fixes it and removes the warning seen when using the old method and NumPy 1.19.